core/src/main/scala/scalafix/rewrite/ProcedureSyntax.scala

```diff
@@ -36,5 +36,6 @@ class ProcedureSyntax[T]() extends Rewrite[T] {
 }
 
 object ProcedureSyntax {
+  val default: ScalafixRewrite = apply()
```

Comment:
I agree. I plan to reorganize the configuration in an upcoming pr and there I will sort out better equality for rewrites.